src/qml/qml.qrc

```diff
@@ -33,6 +33,7 @@
         <file>ProcessingAlgorithmForm.qml</file>
         <file>ProcessingAlgorithmPreview.qml</file>
         <file>processingparameterwidgets/ProcessingParameterWidgetBase.qml</file>
+        <file>processingparameterwidgets/boolean.qml</file>
```

Comment:
I would name those files `ProcessingParameterWidgetBoolean.qml`, `ProcessingParameterWidgetEnum.qml`, ... to find them easier when searching

Reply:
I'm trying to stick to what we do for the feature form editor widgets which also rely on type name without adding a prefix.